cmd/registry/cmd/resolve/resolve.go

```diff
@@ -64,19 +64,13 @@ func Command(ctx context.Context) *cobra.Command {
 
 			log.Printf("Generated %d actions. Starting Execution...", len(actions))
 
-			taskQueue := make(chan core.Task, 1024)
-			for i := 0; i < 64; i++ {
-				core.WaitGroup().Add(1)
-				go core.Worker(ctx, taskQueue)
-			}
-			defer core.WaitGroup().Wait()
-			defer close(taskQueue)
+			for _, a := range actions {
+				log.Printf("Executing action: %s", a)
+				rootCmd := cmd.Root()
+				rootCmd.SetArgs(strings.Split(a, " "))
```

Comment:
I just learned the `strings` package already has a [function](https://pkg.go.dev/strings#Fields) for this. Consider using `strings.Fields(a)` instead.

Reply:
Thanks, done.